packages/react-components/react-table/src/index.ts

```diff
@@ -163,6 +163,7 @@ export type {
   DataGridContextValue,
   DataGridFocusMode,
 } from './DataGrid';
+export { DataGridContextProvider } from './contexts/dataGridContext';
```

Comment:
you can also export the hook to use the context, as long as we're not exporting the context directly it's fine. And in theory to allow recomposition, we should export both